templates/index.html

```diff
@@ -40,7 +40,16 @@ <h2 class="mt-4" id="{{ group }}">{{ group }}</h2>
             <div class="card-body">
               <h5 class="card-title">{{ book[1]['Book Title'] }}</h5>
               <h6 class="card-subtitle mb-2 text-muted">{{ book[1]['Author'] }}<br><small>{{ book[1]['Edition'] }}</small></h6>
-              <a href="{{ book[1]['OpenURL'] }}" class="btn btn-primary stretched-link">Go get it</a>
```

Comment:
I see that the `stretched-link` class is gone. So please add it back. This makes the whole card clickable instead of just the button.

Reply:
Ah, I removed this due to how `stretched-link` was behaving with the link to the Goodreads page. Since the entire card is clickable, it makes the Goodlinks page unclickable as well. Would you know of a possible workaround for this? I tried adjusting the `z-scale` of the two, but no luck.